converter/diag/report.go

```diff
@@ -6,14 +6,47 @@ import (
 
 const Text = ".txt"
 
-// generateTextReport generates a text report for the diagnostics.
-func generateTextReport(writer io.Writer, ds Diagnostics) error {
-	content := ds.Error()
+const criticalErrorFooter = `
+
+A configuration file was not generated due to critical issues. Review the diagnostics above.`
+
+const errorFooter = `
+
+A configuration file was not generated due to errors. Review the diagnostics above.
+
+These may be bypassed which is not recommended for production use.`
```

Comment:
Can this say how to bypass them?

Reply:
```suggestion
You can bypass the errors by using the --bypass-errors flag. Bypassing errors isn't recommended for production environments.`
```

Maybe something like this (the second sentence is passive, but... hard to make active and keep it short)